Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -238,15 +242,22 @@ void GeneralWidget::AddDescriptions()
       QT_TR_NOOP("Uses the main Dolphin window for rendering rather than "
                  "a separate render window.<br><br><dolphin_emphasis>If unsure, leave "
                  "this unchecked.</dolphin_emphasis>");
-  static const char TR_ASPECT_RATIO_DESCRIPTION[] =
-      QT_TR_NOOP("Selects which aspect ratio to use when rendering.<br>"
-                 "Each game can have a slightly different native aspect ratio."
-                 "<br><br>Auto: Uses the native aspect ratio"
-                 "<br>Force 16:9: Mimics an analog TV with a widescreen aspect ratio."
-                 "<br>Force 4:3: Mimics a standard 4:3 analog TV."
-                 "<br>Stretch to Window: Stretches the picture to the window size."
-                 "<br>Custom: For games running with specific custom aspect ratio cheats.<br><br>"
-                 "<dolphin_emphasis>If unsure, select Auto.</dolphin_emphasis>");
+  static const char TR_ASPECT_RATIO_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects which aspect ratio to use when rendering.<br>"
+      "Each game can have a slightly different native aspect ratio.<br>They can vary by "
+      "scene and settings and rarely ever exactly match 4:3 or 16:9."
+      "<br><br><b>Auto</b>: Uses the native aspect ratio"
+      "<br><b>Force 16:9</b>: Mimics an analog TV with a widescreen aspect ratio."
+      "<br><b>Force 4:3</b>: Mimics a standard 4:3 analog TV."
```

Comment:
@AdmiralCurtiss @MayImilae we can resume discussions regarding these tooltips in this PR